keywords/couchbaseserver.py

```diff
@@ -738,12 +738,17 @@ def recover(self, server_to_recover):
         log_info("Setting recover mode to 'delta' for server {}".format(server_to_recover.host))
         data = "otpNode=ns_1@{}&recoveryType=delta".format(server_to_recover.host)
         # Override session headers for this one off request
-        resp = self._session.post(
-            "{}/controller/setRecoveryType".format(self.url),
-            headers={"Content-Type": "application/x-www-form-urlencoded"},
-            data=data
-        )
-
+        count = 0
```

Comment:
if like to try this approach , it would be nice 

from requests.adapters import HTTPAdapter
from requests.packages.urllib3.util.retry import Retry

retry_strategy = Retry(
    total=3,
    status_forcelist=[429, 500, 502, 503, 504],
    method_whitelist=["HEAD", "GET", "OPTIONS"]
)
adapter = HTTPAdapter(max_retries=retry_strategy)
http = requests.Session()
http.mount("https://", adapter)
http.mount("http://", adapter)

response = http.get("https://en.wikipedia.org/w/api.php")

Reply:
what is the advantage compare with current logic? Is this code tested and have you used it anywhere? If yes, can you point me?